### ISSUE TITLE
Any pre-trained model to download ?

### ISSUE THREAD

**Issue by JaywongWang** [jiyanggao/TURN-TAP#2]
Hi, Jiyang. Would you share your trained model so that we can use it for temporal proposal extraction?

**Comment by jiyanggao** [comment]
Hi, currently I don't have trained models, you can just train the model with data I provided.

**Comment by liuhanhui** [comment]
hi，Jiyang,there may be some mistake,the loss is 0.00  in the beginning.
![2631551344008_ pic_hd](https://user-images.githubusercontent.com/24886258/53553789-66020200-3b79-11e9-9b13-0227599f7642.jpg)
